src/main/java/nl/tudelft/ewi/devhub/server/web/resources/AssignmentsResource.java

```diff
@@ -312,7 +312,8 @@ public Response editAssignment(@Context HttpServletRequest request,
                                    @PathParam("assignmentId") long assignmentId,
                                    @FormParam("name") String name,
                                    @FormParam("summary") String summary,
-                                   @FormParam("due-date") String dueDate) {
+                                   @FormParam("due-date") String dueDate,
+                                   @FormParam("release") Boolean release) {
```

Comment:
Maybe `@DefaultValue("false")` is a better idea here. How does the checkbox serialise in a plain html form? Is this serialisable as boolean? ( is the outputted value `"true"` or something else?)